configs/opencv-ovms/grpc_go/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+./grpc-go -i $inputsrc -u 127.0.0.1:$GRPC_PORT -o /tmp/results 2>&1  | tee >/tmp/results/r$cid_count.jsonl >(stdbuf -oL sed -n -e 's/^.*fps: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline$cid_count.log)
```

Comment:
Can fix in a future PR but the IP should use $HOST which is another flag available in the go program